pkg/local_object_storage/metabase/storage_id.go

```diff
@@ -52,3 +52,41 @@ func (db *DB) storageID(tx *bbolt.Tx, addr oid.Address) ([]byte, error) {
 
 	return slice.Copy(storageID), nil
 }
+
+// UpdateStorageIDPrm groups the parameters of UpdateStorageID operation.
+type UpdateStorageIDPrm struct {
+	addr oid.Address
+	id   []byte
+}
+
+// UpdateStorageIDRes groups the resulting values of UpdateStorageID operation.
+type UpdateStorageIDRes struct{}
+
+// SetAddress is an UpdateStorageID option to set the object address to check.
```

Comment:
not use "option"?

Reply:
just want to mention that it is a setter, not an option (functional i mean, like we sometimes do), and recheck that you write it consciously. i have nothing against such wording